content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -25,7 +25,9 @@ And that’s it! It’ll give you any recommendations that apply.
 For example, if you forgot to enable Istio injection (a very common issue), you would get the following 'Info' message:
 
 {{< text syntax=plain snip_id=analyze_all_namespace_sample_response >}}
-Info [IST0102] (Namespace default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection.
+Info [IST0102] (Namespace default) The namespace is not enabled for Istio injection.
```

Comment:
This is requiring the regeneration of the scripts. Will be curious if this is used and may fail the test.

Reply:
Ok, reverted. It's pain to `make gen` 😸